platform/core/src/services/DisplaySetService/DisplaySetService.ts

```diff
@@ -272,7 +275,7 @@ export default class DisplaySetService extends PubSubService {
    * @param settings are settings to add
    * @returns Array of the display sets added.
    */
-  public makeDisplaySetForInstances(
+  private _makeDisplaySetForInstances(
```

Comment:
why are you making this private?

Reply:
Because there is another function public to be called below